packages/admin/admin/src/form/Field.tsx

```diff
@@ -62,12 +62,21 @@ export function Field<FieldValue = any, FieldElement extends HTMLElement = HTMLE
     const shouldShowWarning = passedShouldShowWarning ?? finalFormContext.shouldShowFieldWarning;
     const shouldScrollToField = passedShouldScrollTo ?? finalFormContext.shouldScrollToField;
 
+    const keyRequired = required || validate ? 1 : 0;
+    const keyValidate = validate ? 2 : 3;
```

Comment:
These integer keys are kinda arbitrary. Could we change them to strings? Also, could you please link the example from Final Form here?